lib/omniauth-dsds/lib/user.rb

```diff
@@ -1,22 +1,25 @@
 module Omniauth
   module Dsds
     class User
-      attr_reader :uid, :email, :name, :roles
+      attr_reader :uid, :email, :name, :roles, :organisation_uids
 
       def self.build_from(auth_hash)
+        profile = auth_hash.fetch("profile")
         new(
-          uid:   auth_hash.fetch("profile").fetch("uid"),
-          name:  auth_hash.fetch("profile").fetch("name"),
-          email: auth_hash.fetch("profile").fetch("email"),
-          roles: auth_hash.fetch("roles")
+          uid:   profile.fetch("uid"),
+          name:  profile.fetch("name"),
+          email: profile.fetch("email"),
+          roles: auth_hash.fetch("roles"),
+          organisation_uids: profile.fetch("organisation_uids")
```

Comment:
To maintain the contract this object exposes we should force organisation_uids to be an array (not trust input etc)
Something like:
```
organisation_uids: Array(profile.fetch("organisation_uids"))
```